satpy/etc/composites/ahi.yaml

```
@@ -2,24 +2,11 @@ sensor_name: visir/ahi
 
 modifiers:
   rayleigh_corrected:
-    modifier: !!python/name:satpy.modifiers.PSPRayleighReflectance
-    atmosphere: us-standard
-    aerosol_type: marine_clean_aerosol
-    prerequisites:
-    - wavelength: 0.64
-      modifiers: [sunz_corrected]
-    optional_prerequisites:
-    - satellite_azimuth_angle
-    - satellite_zenith_angle
-    - solar_azimuth_angle
-    - solar_zenith_angle
-
-  no_aerosol_rayleigh_corrected:
     modifier: !!python/name:satpy.modifiers.PSPRayleighReflectance
     atmosphere: us-standard
     aerosol_type: rayleigh_only
     prerequisites:
-    - name: B03
+    - wavelength: B03
```

Why did you change this? B03 isn't a wavelength :-)
It should either remain as `name` or switch to `- wavelength: 0.64`

---

The two rayleigh-based modifiers that were here were essentially the same except for the aerosol type. In general I think we've been specifying bands explicitly in sensor-specific configs so I went with B03 here, but made a typo.

```suggestion
    - name: B03
```